jme3-plugins/src/gltf/java/com/jme3/scene/plugins/gltf/ext/JME_speaker/SpeakerExtensionLoader.java

```diff
@@ -0,0 +1,101 @@
+ *   may be used to endorse or promote products derived from this software
+ *   without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jme3.scene.plugins.gltf.ext.JME_speaker;
+
+
+import com.jme3.asset.AssetManager;
+import com.jme3.audio.AudioData.DataType;
+import com.jme3.plugins.json.JsonElement;
+import com.jme3.plugins.json.JsonObject;
+import com.jme3.audio.AudioNode;
+import com.jme3.scene.Node;
+import com.jme3.scene.plugins.gltf.ExtensionLoader;
+import com.jme3.scene.plugins.gltf.GltfLoader;
+
+/**
+ * An extension for the GLTF loader that loads speaker nodes.
+ * (This extension requires the jme-extras addon for blender)
+ */
+public class SpeakerExtensionLoader implements ExtensionLoader {
+
+    public SpeakerExtensionLoader(){}
+
+    @Override
+    public Object handleExtension(GltfLoader loader, String parentName, JsonElement parent, JsonElement extension, Object input) {
+        if (input instanceof Node) {
+            JsonObject jsonObject = extension.getAsJsonObject();
+
+            final float volume = jsonObject.getAsJsonPrimitive("volume").getAsFloat();
+            final float pitch = jsonObject.getAsJsonPrimitive("pitch").getAsFloat();
+            final float attenuation = jsonObject.getAsJsonPrimitive("attenuation").getAsFloat(); // unused
+            final float distanceMax = jsonObject.getAsJsonPrimitive("distance_max").getAsFloat();
+            final float distanceReference = jsonObject.getAsJsonPrimitive("distance_reference").getAsFloat();
+            final float volume_min = jsonObject.getAsJsonPrimitive("volume_min").getAsFloat(); // unused
+            final float volume_max = jsonObject.getAsJsonPrimitive("volume_max").getAsFloat(); // unused
+            final float angleOuterCone = jsonObject.getAsJsonPrimitive("angle_outer_cone").getAsFloat();
+            final float angleInnerCone = jsonObject.getAsJsonPrimitive("angle_inner_cone").getAsFloat();
+            final float outerConeVolume = jsonObject.getAsJsonPrimitive("outer_cone_volume").getAsFloat(); // unused
+            final String soundPath = jsonObject.getAsJsonPrimitive("sound_path").getAsString();
```

Comment:
Parameters are named in camel case in the rest of the Gltf extensions so I would suggest keeping the same naming convention here as well. (i.e. `distance_max` -> `distanceMax`,...)